CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(cmake_common_scripts VERSION 0.1.0 LANGUAGES CXX)
+include(cmake/cmake_tools.cmake)
+include(cmake/code_coverage.cmake)
+
+configure_package()
+
+install(FILES
+  "${CMAKE_CURRENT_LIST_DIR}/cmake/cmake_tools.cmake"
+  "${CMAKE_CURRENT_LIST_DIR}/cmake/code_coverage.cmake"
+  DESTINATION lib/cmake/${PROJECT_NAME})
```

Comment:
Consider using [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html).

Reply:
Is this cross-platfrom?